src/sidebar/services/test/frame-sync-test.js

```diff
@@ -663,6 +664,26 @@ describe('FrameSyncService', () => {
       assert.calledWith(fakeStore.selectAnnotations, ['id1', 'id2', 'id3']);
       assert.calledWith(fakeStore.selectTab, 'annotation');
     });
+
+    it('does not request keyboard focus if `focusFirstInSelection` is false', () => {
+      fakeStore.findIDsForTags.returns(['id1', 'id2', 'id3']);
+      emitGuestEvent(
+        'showAnnotations',
+        ['tag1', 'tag2', 'tag3'],
+        false /* focus */
+      );
+      assert.notCalled(fakeStore.setAnnotationFocusRequest);
+    });
+
+    it('requests keyboard focus for first annotation in selection', () => {
```

Comment:
Perhaps add a test that demonstrates the behavior if the first/some of the IDs provided to `showAnnotations` weren't found by `findIDsForTags`?

Reply:
I added a test case for when `findIDsForTags` returns no IDs, since that case results in different logic being executed.